source/options.tsx

```diff
@@ -70,10 +70,15 @@ async function getTokenScopes(personalToken: string): Promise<string[]> {
 	return scopes;
 }
 
+function expandTokenSection(): void {
+	select('details#token')!.open = true;
+}
+
 async function validateToken(): Promise<void> {
 	reportStatus({});
 	const tokenField = select('input[name="personalToken"]')!;
 	if (!tokenField.validity.valid || tokenField.value.length === 0) {
+		expandTokenSection();
```

Comment:
## Without token or invalid token: expanded

<img width="555" alt="Screenshot 2" src="https://user-images.githubusercontent.com/1402241/221105682-7b1b6384-a8ea-4584-8aa9-06225ac07086.png">

## With valid token: collapsed

<img width="555" alt="Screenshot 1" src="https://user-images.githubusercontent.com/1402241/221105664-39be2e57-df4c-4a77-b898-b27b62f203e4.png">

## Demo of a page reload after adding a token

https://user-images.githubusercontent.com/1402241/221105827-cc948266-c68b-489a-887b-ab61200f6d89.mov

## Demo of section expansion after switching to a GHE host without token

https://user-images.githubusercontent.com/1402241/221106130-2cb4c036-26ad-4dce-8cb3-941c14a46887.mov